File: CloudFlow/FrontEnd-App/src/app/shared/model/nomenclator/save-nomenclator-value-response.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { NomenclatorAttributeModel } from "./nomenclator-attribute.model";

@JsonObject
export class SaveNomenclatorValueResponseModel {
	
	public static readonly STATUS_SUCCESS: string = "SUCCESS";
	public static readonly STATUS_ERROR: string = "ERROR";

	@JsonProperty("status", String)
	public status: string = null;

	@JsonProperty("messages", [String], true)
	public messages: string[] = [];
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/nomenclator/SaveNomenclatorValueResponseModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.nomenclator;

import java.util.List;

public class SaveNomenclatorValueResponseModel {

	private SaveNomenclatorValueResponseStatus status;
	private List<String> messages;
	
	public static enum SaveNomenclatorValueResponseStatus {
		SUCCESS,
		ERROR
	}
	
	public SaveNomenclatorValueResponseStatus getStatus() {
		return status;
	}
	
	public void setStatus(SaveNomenclatorValueResponseStatus status) {
		this.status = status;
	}
	
	public List<String> getMessages() {
		return messages;
	}
	
	public void setMessages(List<String> messages) {
		this.messages = messages;
	}
}



